task01-html-css/characters.css

```diff
@@ -0,0 +1,71 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary short description for the file
+ */
+
+.grid-container {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    gap: 20px;
+
+    width: 940px;
+    height: 279.65px;
+    overflow-x: scroll;
+
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+}
+
+.grid-item {
+    position: relative;
+    /* display: flex;
+    align-items: flex-end;
+    text-align: center; */
+
+    width: 219.74px;
+    height: 279px;
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
```

Comment:
Same as before, try not to follow the CSS code, we only see the fonts, color, sizes, distances...
Give flex a try on this approach, for example:
`display: flex;
flex-direction: column`

You will notice a similar result with less lines of code. :)